action.yml

```diff
@@ -19,6 +19,12 @@ inputs:
   mask-aws-account-id:
     description: "Whether to set the AWS account ID for these credentials as a secret value, so that it is masked in logs. Valid values are 'true' and 'false'. Defaults to true"
     required: false
+  role-to-assume:
+    description: "Use the provided credentials to assume a role rather than persisting the credentials directly"
```

Comment:
What I meant by "persisting" was "passing the values along to later steps via outputs".

Maybe
`Use the provided credentials to assume a Role and output the assumed credentials for that Role rather than the provided credentials.` ?